fe/fe-core/src/main/java/org/apache/doris/planner/SingleNodePlanner.java

```diff
@@ -1902,7 +1902,7 @@ private void materializeTableResultForCrossJoinOrCountStar(TableRef tblRef, Anal
      * @param analyzer
      */
     private void materializeBaseTableRefResultForCrossJoinOrCountStar(BaseTableRef tblRef, Analyzer analyzer) {
```

Comment:
I think this method name should be changed to a more common name?
like: `materializeSlotForEmptyMaterializedTableRef()`?
And add examples in comment to explain why we call it.